### ISSUE TITLE
implement initial setting of phrase for `organisation_name`

### ISSUE THREAD

**Issue by Pfiffikus** [eprints/eprints3.4#113]
As done by `sub config_core@epadmin` for `archive_name` a useful or user-controlled definition has to be set for the request of `accessibility/index.xpage`.

**Comment by drn05r** [comment]
As you have stated organisation_name is set by running "epadmin create" and sets this at the same time as setting the archive_name.  There is no initial setting for archive_name so I do not believe it would be appropriate to have one for organisation_name.  Unlike other phrases, there is no way to set an initial value that would be accurate, so it is best not to have an initial value at all.  This ensure a "phrase not defined" message is displayed and the repository administrator sets a value.

It may be worth adding to the notice at the top of the initial Accessibility page so it says:

_"This page is intended as a generic accessibility statement for EPrints and should be amended as required before deployment of your EPrints repository.  In particular creating a phrase for organisation_name if this is not already defined for ARCHIVE_NAME."_
